### PR TITLE
Update openmetadata helm chart to address issue #209 and #210

### PR DESCRIPTION
**Pull request by preetsshah**
### Describe your changes :

This PR addresses issue [#209](https://github.com/open-metadata/openmetadata-helm-charts/issues/209) and [#210](https://github.com/open-metadata/openmetadata-helm-charts/issues/210).

- Updated the initContainer settings to use `resources` from values.yaml.
- Added `automountServiceAccountToken` in values.yaml for OMD pod/deployment. Updated required files accordingly.

#
### Type of change :

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developer) document.
- [ ] I have performed a self-review of my own. 
- [ ] I have tagged my reviewers below.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@akash-jain-10

### CODE REVIEW

**Comment by harshach** [comment]
@preetsshah are you planning on turning that into a draft to a PR. Any pending changes here?

**Comment by tutte** [comment]
Thank you for you contribution @preetsshah , it looks good from our side :)
Feel free to set it ready for review and we'll merge it and issue a new release including your changes.

**Comment by preetsshah** [comment]
Many thanks @tutte! Moving the PR to ready for review.

File: charts/openmetadata/values.yaml [hunk 1]
```diff
@@ -362,17 +363,17 @@ extraVolumeMounts: []
...
-#   memory: 128Mi
-# requests:
-#   cpu: 100m
-#   memory: 128Mi
+  limits:
+    cpu: 1
+    memory: 2048Mi
+  requests:
+    cpu: 500m
+    memory: 256Mi
```
Comment (akash-jain-10) [inline charts/openmetadata/values.yaml:376]: We do not want to enable these limits and requests! Let's keep it in comments here!

Comment (preetsshah) [reply charts/openmetadata/values.yaml:376]: Acknowledged, updating the PR.

**Review by akash-jain-10 — APPROVED** [review]

**Pull request merged**